package.json

```diff
@@ -31,6 +31,7 @@
     "emit-then": "^2.0.0",
     "event-promise": "^0.0.1",
     "flatmap": "^0.0.3",
+    "lodash": "^4.17.15",
```

Comment:
I don't think we really need lodash for this

Reply:
I used lodash for get & set strings

Reply:
yes, you can do it without using that lib, it's pretty big for this basic use case

Reply:
hmm, well, will rewrite later